### DISCUSSION TITLE
Type: Miscellaneous; New wallet address

### DISCUSSION THREAD

**Discussion by Heuer212** [Chia-Network/chia-blockchain#4138] [Support]
I have two incomings of 0.75 which is shown on GUI, and this two incomings have been confirmed at a block height. But my wallet address balance didn't receive them even after four days. They were sent to a new wallet address rather than my first wallet address. I have searched the address on Chia Explorer and it shows the two incomings' type is Miscellaneous not Farmer Reward.

1. What is Miscellaneous type of incoming?
2. How/Where can I find them?
3. And what is the 'new address' button on GUI? Will all XCHs sent to different wallet addresses be displayed in the wallet balance?

Thanks for any help!

**Comment by DrHou83** [comment]
Same issue..following

**Reply by DrHou83** [reply]
how do you find your "first wallet address" btw

**Reply by Heuer212** [reply]
use CLI
chia keys show
you‘ll have ur first wallet address at the end of the information shown

**Reply by DrHou83** [reply]
ok great thank you. so my first wallet address matches my  xch_target_address in my config which I guess is good. its just the random receive address with 0.78 chia in it Im not sure about :)

**Comment by earthknow1** [comment]
I've got the same issue

**Reply by earthknow1** [reply]
I have one incoming 1.75 xch

**Comment by Sagittarius** [comment]
I got the same issue 2. What's that mean?

**Comment by yangyang8921** [comment]
I've got the same issue

**Comment by yangyang8921** [comment]
Wallet receiving address don't know whether or not updated automatically, but farming address didn't change, had the two 0.75, money not to account, can block chain browser to check the new address 1.5 COINS, but the wallet can't find, replace how back? In addition
Has been in doubt, receiving address would become words explodes the block, farming as can address to and receive it, must keep has been to the two address?

**Comment by nooksquire** [comment]
Got the same issue. 

I saw someone in a closed issues say that it has to do with UTXO model (I think he is part of the CHIA team). 
Don't fully understand it and why it would show in my wallet if its someone elses "change". 

#5220 #5216 If you are interested. 

I am just grasping at straws though, I am lost too.

**Reply by Heuer212** [reply]
Thx. Seems like the Chia team will explain what is change transaction more clear in the future.

**Comment by BrandtH22** [comment]
@Heuer212 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
